components/gfx/font_template.rs

```diff
@@ -84,12 +90,14 @@ impl<'a> From<&'a FontStyleStruct> for FontTemplateDescriptor {
 /// font instance handles. It contains a unique
 /// FontTemplateData structure that is platform specific.
 pub struct FontTemplate {
-    identifier: FontIdentifier,
-    descriptor: Option<FontTemplateDescriptor>,
-    weak_ref: Option<Weak<FontTemplateData>>,
-    // GWTODO: Add code path to unset the strong_ref for web fonts!
-    strong_ref: Option<Arc<FontTemplateData>>,
-    is_valid: bool,
+    pub identifier: FontIdentifier,
```

Comment:
Are these fields intentionally `pub` and not `pub(crate)`?

Reply:
Yes, I think these have to be pub because they are used by out-of-crate tests unfortunately.